tmf_template.altis/briefing/briefing_nato.sqf

```diff
@@ -0,0 +1,83 @@
+/* ===============================================
+	GENERAL BRIEFING NOTES
+	 - Uses HTML style syntax. All supported tags can be found here - https://community.bistudio.com/wiki/createDiaryRecord
+	 - For images use <img image='FILE'></img> (for those familiar with HTML note it is image rather than src).
+	 - Note that using the " character inside the briefing block is forbidden use ' instead of ".
+*/
+
+/* ===============================================
+	SITUATION
+	 - Outline of what is going on, where we are we and what has happened before the mission has started? This needs to contain any relevant background information.
+	 - Draw attention to friendly and enemy forces in the area. The commander will make important decisions based off this information.
+	 - Outline present weather conditions, players will typically assume that it is daylight with sunny weather.
+*/
+
+private _situation = ["diary", ["Situation","
+<br/>
+*** Insert general information per the above description.***
+<br/><br/>
+<font size='18'>ENEMY FORCES</font>
+<br/>
+*** Provide information on enemy forces.***
+<br/><br/>
+
+<font size='18'>FRIENDLY FORCES</font>
+<br/>
+*** Detail friendly forces that are external to the playable force. ***
+
+"]];
+
+/* ===============================================
+	MISSION
+	 - Describe any objectives that the team is expected to complete.
+	 - Summarize(!) the overall task. This MUST be short and clear.
+*/
+
+private _mission = ["diary", ["Mission","
+<br/>
+*** Provide a short summary of the mission objectives here. ***
+"]];
+
+/* ===============================================
+	EXECUTION
+	 - Provide an outline as to what the commander of the player's command might give.

```

Comment:
`Provide an outline as to what the commander of the player's command might give.`
Who would write such nonsense?
`Provide an outline of intent for the commander to use as inspiration, alongside any extra information on how the platoon should move, if external support is available and lastly if there are extra tasks that should be considered.`